test/intergration/provider-s3.js

```diff
@@ -58,4 +58,21 @@ describe('S3 Provider', () => {
 
     it('to delete a file', () => manager.deleteFile(testLocation, testFileName));
   });
+
+  describe('should initialize S3 provider without options', () => {
```

Comment:
More of a tip, than some mistake, but in `describe(` you usually want to write something like the name of the tool or a class that you test and then in the `it(` you write what it should/should not do. This works so, that when running a test, the message is taken from the `describe(` function and then added before each message inside teh `it(` function. So if you have, for example, the following test:
```
describe('My laptop', () => {
  it('should be able to charge', () => {});
  it('should be able to write code for me', () => {});
  it('should be able to look for pictures of cats when I'm sad', () => {});
})
```
it will result in the following console output:
```
✓ My laptop should be able to charge
✓ My laptop should be able to write code for me
✓ My laptop should be able to look for pictures of cats when I'm sad
```
You see? So it takes that string from the `describe(` function and appends it to the string from `it` function and altogether, it is combined into a readable test case.

In your case, it will be 
`should initialize S3 provider without options should work without options`

Not very clear, right?
So, if you can, please adjust it to make it look perfect.

Reply:
Very well, thank you for enlightening me on this subject, 
I will rectify this immediately.